README.md

```diff
@@ -35,3 +44,19 @@ If you're using Lumen, add the following line to your `bootstrap/app.php` file:
 ```php
 $app->register(PodPoint\I18n\Providers\CountriesServiceProvider::class);
 ```
+
+## Changelog
+
+Please see [CHANGELOG](CHANGELOG.md) for more information on what has changed recently.
+
+## License
+
+The MIT License (MIT). Please see [License File](LICENCE.md) for more information.
```

Comment:
Licence not License here

license is the verb for the UK whereas America uses license for both :D I've learnt something while looking at this: https://www.grammar-monster.com/easily_confused/licence_license.htm